### ISSUE TITLE
FP: 500 response when trying to get status on non-existing flow

### ISSUE THREAD

**Issue by hpl002** [stacc/stacc-flow-community#43]
**Module and version**
FP `version=10.10.1093`

**Describe the bug**
FP gets an internal err when trying to get status on non-existing flow. 

**To reproduce**

`curl --location --request GET 'http://localhost:1337/api/flows/invalid-url/status'  
{"title":"An error occurred while processing your request","detail":"Check application logs for details","traceId":"572ea215-bea1-42e1-89ed-88e3c443e668","status":500,"message":"Check application logs for details"}`
 
**Expected behavior**
should get a 404 response 

```
flow-process-7f45d88744-5rl59 flow-process {"message":["Caught error from API, traceId: c4e3c936-87d6-4076-83bc-4921d4e80b86, details: ","TypeError: Cannot destructure property 'flow' of 'statusResponse' as it is undefined.\n    at /app/src/api/flows.js:106:13\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:94:5)"],"level":"error"}
flow-process-7f45d88744-5rl59 flow-process {"message":"GET /api/flows/69848f9b-f195-4f4f-a792-baafbf62014/status 500 214 - 7.813 ms\n","level":"error"}
```

**Comment by Anders-eide-stacc** [comment]
Good catch, I've put it on my task list

**Comment by Anders-eide-stacc** [comment]
Fixed in v11.2 and v10.10 now.
